packages/ui-components/src/components/QrCode/useQrCodeCanvas.tsx

```diff
@@ -66,34 +60,34 @@ export const drawQrCodeCanvas = async (
   if (!canvas) {
     return null;
   }
-  canvas.setAttribute('width', `${SCALED_CANVAS_SIZE}`);
-  canvas.setAttribute('height', `${SCALED_CANVAS_SIZE}`);
+  canvas.setAttribute('width', `${CANVAS_WIDTH}`);
+  canvas.setAttribute('height', `${CANVAS_HEIGHT}`);
 
   const ctx: CanvasRenderingContext2D | null = canvas.getContext('2d');
   if (!ctx) {
     return null;
   }
   // Background
   ctx.fillStyle = 'white';
-  ctx.fillRect(0, 0, SCALED_CANVAS_SIZE, SCALED_CANVAS_SIZE);
+  ctx.fillRect(0, 0, CANVAS_WIDTH, CANVAS_HEIGHT);
 
   // Add header text
-  const textY = 60 * DOWNLOAD_IMAGE_SCALE; // Make sure to avoid the QR code "quiet zone"
+  const textY = 250; // Make sure to avoid the QR code "quiet zone"
+  const textX = 75;
   ctx.fillStyle = 'black';
-  ctx.font = `${39 * DOWNLOAD_IMAGE_SCALE}px monospace`;
-  ctx.textAlign = 'center';
-  ctx.fillText(humanReadableId, SCALED_CANVAS_SIZE / 2, textY);
+  ctx.font = `105px monospace`;
+  ctx.fillText(humanReadableId, textX, textY);
 
   // Add qr code
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
   const qrDataUrl = await toDataURL(qrCodeContents, {
-    width: SCALED_CODE_SIZE,
-    height: SCALED_CODE_SIZE,
+    width: CANVAS_HEIGHT,
+    height: CANVAS_HEIGHT,
   });
   const img = new Image();
-  const codeX = (SCALED_CANVAS_SIZE - SCALED_CODE_SIZE) / 2;
-  const codeY = 65 * DOWNLOAD_IMAGE_SCALE; // Make sure to avoid the QR code "quiet zone"
+  const codeX = 900;
+  const codeY = 0; // Make sure to avoid the QR code "quiet zone"
```

Comment:
Seems like there are a few repeated numbers in here. Would be good to make them variables like `CANVAS_WIDTH` etc are, so that if we end up changing any, they are just changed in 1 place and we don't accidentally forget some.